README.adoc

```diff
@@ -6,6 +6,89 @@ image:https://codecov.io/gh/codeready-toolchain/toolchain-common/branch/master/g
 
 This repo is for controllers, libs, scripts, make files, etc to be shared between host and member operators.
 
+== Setting Up and Connecting Host and Member Clusters
+
+In a new terminal, execute the following commands
+```
+$ cd $GOPATH/github.com/codeready-toolchain/host-operator
+$ minishift start --profile host
+$ make up-local
+^C
+```
+
+```
+$ cd $GOPATH/github.com/codeready-toolchain/member-operator
+$ minishift start --profile member
+$ make up-local
+^C
+$ make add-member-to-host
+$ make add-host-to-member
+$ make up-local
+```
+
+Now that we are setup, we can add resources and watch the logs on the member-operator! To do so, open a new terminal and run the following commands
+```
+$ oc project toolchain-host-operator
+$ oc apply -f '/path/to/masteruserrecord.yaml'
+$ oc project toolchain-member-operator
+$ oc apply -f '/path/to/useraccount.yaml'
+```
+
+**Example masteruserrecord.yaml:**
+```
+apiVersion: toolchain.dev.openshift.com/v1alpha1
+kind: MasterUserRecord
+metadata:
+  name: example
+spec:
+  disabled: false
+  deprovisioned: false
+  userID: 86505192-a386-11e9-ad56-525400ad2b23
+  userAccounts:
+  - targetCluster: member-192-168-42-61-8443
+    syncIndex: 86505192-a386-11e9-ad56-525400ad2b23
+    spec:
+      nsLimit: admin
+      userID: 86505192-a386-11e9-ad56-525400ad2b23
+      nsTemplateSet:
+        tierName: basic
+        namespaces:
+        - type: ide
+          revision: abcdef
+        - type: cicd
+          revision: abcdef
+status:
+  conditions:
+  - type: Ready
+    status: True
+  userAccounts:
+  - targetCluster: target-member-cluster
+    syncIndex: 86505192-a386-11e9-ad56-525400ad2b23
+```
+
+**Example useraccount.yaml:**
+```
+apiVersion: toolchain.dev.openshift.com/v1alpha1
+kind: UserAccount
+metadata:
+  name: example
+spec:
+  userID: 86505192-a386-11e9-ad56-525400ad2b23
+  disabled: false
+  nsLimit: admin
+  nsTemplateSet:
+    tierName: basic
+    namespaces:
+    - type: ide
+      revision: abcdef
+    - type: cicd
+      revision: abcdef
+status:
```

Comment:
I guess one last thing, when we use `oc apply` to create CR.  The CR file should not contain `status` field.  I guess, it will be ignored.  `status` is populated by reconciler.  Please remove `status` from both example yaml file.

Reply:
@nurali-techie Removed - thanks for letting me know!